x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -65,11 +65,11 @@ environment variables to run inside of the container.
 
 The following actions are possible and grouped based on the actions.
 
-* Elastic Agent Fleet Enrollment
+* Elastic Agent Fleet Server Enrollment
```

Comment:
This is one of the areas I started to stumble over the Fleet Server naming. We call the flag `FLEET_ENROLL` as there are also more flags with fleet-server below. And from a pure user perspective the enrollment happens into Fleet. That the service behind it is fleet-server is only relevant for debugging.

Reply:
I see what you mean here, the difference between fleet and fleet-server is a bit blurred. 
As a concepts we have "Fleet" that has two distinct part a server and a ui part. I think _FLEET_ENROLL_ is fine in this context because you are enrolling into the "system" which is fleet.

But as anything, I think our users will help us to refine the settings or messages.